src/common/internet.rs

```diff
@@ -51,13 +54,14 @@ pub enum InetAddr {
     IPv6(Ipv6Addr),
     #[cfg(feature = "tor")]
     Tor(TorPublicKeyV3),
+    TorV2(OnionAddressV2),
 }
 
 impl InetAddr {
     #[cfg(feature = "tor")]
-    pub const UNIFORM_ADDR_LEN: usize = TORV3_PUBLIC_KEY_LENGTH;
+    pub const UNIFORM_ADDR_LEN: usize = TORV3_PUBLIC_KEY_LENGTH; //32usize
     #[cfg(not(feature = "tor"))]
-    pub const UNIFORM_ADDR_LEN: usize = 32;
+    pub const UNIFORM_ADDR_LEN: usize = 32; //Todo: This seems redundant, can be removed
```

Comment:
The reason this seemed odd to me is `TORV3_PUBLIC_KEY_LENGTH` is also 32 bytes. So tor or non-tor, this will always end up being 32 bytes. Unless I am missing something.

Reply:
It is, but there is semantic difference. In the first case, when Tor support is enabled, we stick to the TOR key, in the other case, we use the longest data. If in the future we will have more data structures, it will help not to miss the need in constant update

Reply:
Right. It's better to keep it future extensibility.